numpy/_core/_add_newdocs.py

```diff
@@ -3092,7 +3092,6 @@
     :pep:`585` : Type hinting generics in standard collections.
     numpy.typing.NDArray : An ndarray alias :term:`generic <generic type>`
                         w.r.t. its `dtype.type <numpy.dtype.type>`.
-
```

Comment:
This whitespace fix seems out-of-scope for this PR.

Reply:
Gotcha, that was just a leftover from adding an entry for `npt.Array` and then removing it, rather than reverting it.